.mergify.yml

```diff
@@ -12,9 +13,9 @@ pull_request_rules:
         add: ["auto merged"]
   - name: Automatically merge PRs from maintainers on CI success and review
     conditions:
-    - status-success=continuous-integration/travis-ci/pr
-    - "label=ready to merge"
-    - "author=@oss-approvers"
+      - status-success=continuous-integration/travis-ci/pr
+      - "label=ready to merge"
+      - "author=@oss-approvers"
```

Comment:
TODO later: Change this to the right team. Or remove entirely. We can automate releases to ECR in Travis on tags/pushes to master.